2015/presentations/with modern_peripherals: Python and Flask Piotr Dyba/text.md

```diff
@@ -0,0 +1,210 @@
+# with modern_peripherals: Python and Flask - Piotr Dyba
+
+##Introduction
+
+Auto-scrolling sites, glance-following ads, and gesture friendly web pages are 
+coming! Over the last few years three products emerged that enable interaction 
+with computer in a new way: Myo Armband, Leap Motion Controller and EyeTribe. 
+The integration between the aforementioned products, other devices and web
+apps is based on the web sockets protocol. Lets start with really short 
+introduction to Web Sockets, after that I’ll get through implementation, 
+wrappers and an imaginative app for each of the new devices.
```

Comment:
After reading the entire, I'd say it should be imaginary, not sample nor imaginative